test/test_torch.py

```diff
@@ -12491,6 +12711,14 @@ def test_lerp_mem_overlap(self):
         with self.assertRaisesRegex(RuntimeError, 'single memory location'):
             start.lerp_(end, weight)
 
+    @unittest.expectedFailure
```

Comment:
Fails currently only because  `inplace_op` below is not declared

Reply:
Fixed